entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/impl/EntityRecordService.java

```diff
@@ -343,4 +368,62 @@ else if (fieldValuePrimaryObject != null) {
     public void dropRepository(){
     	this.entityRecordRepository.dropCollection();
 	}
+
+    private void setEntityAggregation(EntityRecord entityRecord, String entityId) {
+        // set isAggregatedBy on Entity
+        Aggregation isAggregatedBy = new AggregationImpl();
+        isAggregatedBy.setId(entityId + "#aggregation");
+        isAggregatedBy.setCreated(new Date());
+        isAggregatedBy.setModified(new Date());
+        isAggregatedBy.setRecordCount(1);
+        isAggregatedBy.setAggregates(Arrays.asList(getEuropeanaAggregationId(entityId), getDatasourceAggregationId(entityId)));
+
+        entityRecord.getEntity().setIsAggregatedBy(isAggregatedBy);
+    }
+
+    private void setEuropeanaMetadata(String entityId, EntityRecord entityRecord) {
+        String europeanaProxyId = entityId + "#proxy_europeana";
+
+        Aggregation europeanaAggr = new AggregationImpl();
+        europeanaAggr.setId(getEuropeanaAggregationId(entityId));
+        //TODO: move to constants file
+        europeanaAggr.setRights("https://creativecommons.org/publicdomain/zero/1.0/");
+        europeanaAggr.setCreated(new Date());
+        europeanaAggr.setModified(new Date());
+        europeanaAggr.setSource("www.europeana.eu");
```

Comment:
source field  (dc:source) has a different semantic meaning in Aggregation objects, being used for depictions only. We should not set this field in the aggregation